packages/react-components/src/index.js

```diff
@@ -0,0 +1,90 @@
+import AcceptTermsPrompt from './components/AcceptTermsPrompt/AcceptTermsPrompt';
```

Comment:
This file supports the webpack aliasing in storybook and is based off of the [index file at the root of `react-components`](https://github.com/department-of-veterans-affairs/component-library/blob/42326ff2d08fe8fac56bb91b14ef6b2aacd2166e/packages/react-components/index.js#L1). I tried simplifying that file to just `export * from './src'` but I couldn't get a passing build for `core`.

Reply:
:thinking: Interesting. This is worth a follow-up later. not maintaining two `index.js` files would be ideal